web-api/tests/document_candidates.rs

```diff
@@ -19,10 +19,9 @@ use reqwest::{Client, StatusCode, Url};
 use serde::Deserialize;
 use serde_json::{json, Value};
 use xayn_integration_tests::{send_assert, send_assert_json, test_app, UNCHANGED_CONFIG};
-use xayn_test_utils::error::Panic;
 use xayn_web_api::Ingestion;
 
-async fn ingest(client: &Client, url: &Url) -> Result<(), Panic> {
+async fn ingest(client: &Client, url: &Url) -> Result<(), anyhow::Error> {
```

Comment:
:art: reuse via import, similarly for the other occurences
```suggestion
use anyhow::Error;

async fn ingest(client: &Client, url: &Url) -> Result<(), Error> {
```